src/include/migraphx/op/rnn_last_hs_output.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_RNN_LAST_HS_OUTPUT_HPP
+#define MIGRAPHX_GUARD_OPERATORS_RNN_LAST_HS_OUTPUT_HPP
+
+#include <migraphx/op/rnn_last_output.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+struct rnn_last_hs_output : rnn_last_output<rnn_last_hs_output>
+{
+    rnn_last_hs_output() {}
+    rnn_last_hs_output(rnn_direction dirct) : rnn_last_output(dirct) {}
```

Comment:
This shouldn't take the direction. The direction is only needed for `rnn_var_sl_last_output` operator.